# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1381782427 -7200
#      Mon Oct 14 22:27:07 2013 +0200
# Node ID b4783277bd115f9f95c16a5d17b271f7e1f09612
# Parent  57750dc54ab675422b1586298dee9316987e2e65
*octave-qscintilla.cc: fix edit function at cursor by shortcut Ctrl-E

diff --git a/libgui/src/m-editor/octave-qscintilla.cc b/libgui/src/m-editor/octave-qscintilla.cc
--- a/libgui/src/m-editor/octave-qscintilla.cc
+++ b/libgui/src/m-editor/octave-qscintilla.cc
@@ -71,17 +71,17 @@ octave_qscintilla::context_help_doc (boo
   if (get_actual_word ())
     contextmenu_help_doc (documentation);
 }
 
 // call edit the function related to the current word
 void
 octave_qscintilla::context_edit ()
 {
-  if (hasSelectedText ())
+  if (get_actual_word ())
     contextmenu_edit (true);
 }
 
 // call edit the function related to the current word
 void
 octave_qscintilla::context_run ()
 {
   if (hasSelectedText ())
